src/pages/UserHome.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import firebase from 'firebase';
+import {
+  Button,
+  Container,
+  Grid,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    paddingLeft: theme.spacing(6),
+  },
+  bannerRoot: {
+    backgroundColor: 'white',
+    marginTop: theme.spacing(4),
+    minHeight: '225px',
+    paddingLeft: theme.spacing(3),
+    paddingTop: theme.spacing(4),
+  },
+  browseButton: {
+    marginTop: theme.spacing(3),
+  },
+  addButton: {
+    textAlign: 'right',
+  },
+  scholarshipsAdded: {
+    marginTop: theme.spacing(6),
+  },
```

Comment:
There's a lot of spacing issues here. But let's have @Jobhdez fix them after this gets merged as part of the styling fixes.

Reply:
sounds good